src/App.js

```diff
@@ -1,22 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
 
+import { useEffect, useState } from 'react';
+import getPokemon from './services/fetch-utils';
+import PokemonList from './PokemonList';
+import Spinner from './Spinner';
+
 function App() {
+  const [pokemon, setPokemon] = useState([]);
+  const [query, setQuery] = useState('char');
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    async function load() {
+      setLoading(true);
+      const response = await getPokemon(query);
+      setPokemon(response.data.results);
+      setLoading(false);
+    }
+    load();
+  }, []); //eslint-disable-line
+
+  async function handleSearch(e) {
+    e.preventDefault();
+
+    async function load() {
```

Comment:
because you're using the same load function here and in the useEffect, you can pull it out of both to declare and then call it inside of both. it will cut down on the redundancy.